packages/jest/src/jest.js

```diff
@@ -7,4 +7,20 @@
  * @flow
  */
 
-export {default} from 'jest-cli';
+import cli from 'jest-cli';
+
+const obj = {
+  ...jest,
+  afterAll,
+  afterEach,
+  beforeAll,
+  beforeEach,
+  describe,
+  expect,
+  fdescribe: describe.only,
+  it: test,
+  test,
+  xdescribe: describe.skip,
+};
```

Comment:
this seems to be missing `fit`, `xit`, and `xtest`. (full list is at https://jestjs.io/docs/en/api)